lib/suites/LinkedDataSignature.js

```diff
@@ -193,12 +204,14 @@ module.exports = class LinkedDataSignature {
   async verify(framed, options) {
     options = Object.assign({}, options || {});
 
+    // TODO: This only supports one proof object at a time.
```

Comment:
I believe this comment can be remove, multiple proofs are properly handled in jsonld-signatures.js.  Each `LinkedDataSignature` is verified one at a time at this level.